tests/expectations/tests/libclang-9/class.rs

```diff
@@ -16,12 +16,12 @@ impl<T> __IncompleteArrayField<T> {
         __IncompleteArrayField(::std::marker::PhantomData, [])
     }
     #[inline]
-    pub unsafe fn as_ptr(&self) -> *const T {
-        ::std::mem::transmute(self)
+    pub fn as_ptr(&self) -> *const T {
+        self as *const Self as *const T
```

Comment:
Is `Self` stable in rust 1.0? Probably worth just doing `self as *const _ as *const T`.

Reply:
I tried compiling on rust 1.0.0 and it compiled but I can replace with `_`